virttest/utils_test/qemu.py

```diff
@@ -146,6 +147,32 @@ def clear_win_driver_verifier(session, vm, timeout=300):
     return vm.reboot(session)
 
 
+def setup_runlevel(params, session):
+    """
+    Setup the runlevel in guest.
+
+    :param params: Dictionary with the test parameters.
+    :param session: VM session.
+    """
+    expect_runlevel = params.get("expect_runlevel", "3")
+    cmd = "runlevel"
+
+    if utils_misc.wait_for(lambda: session.cmd_status(cmd) == 0, 15):
+        ori_runlevel = session.cmd(cmd)
+    else:
+        ori_runlevel = "0" 
+
```

Comment:
The following is a simplicity way, Just suggestion
ori_runlevel = "0"
if utils_misc.wait_for(lambda: session.cmd_status(cmd) == 0, 15):
    ori_runlevel = session.cmd(cmd)